governance/program/tests/process_complete_proposal.rs

```diff
@@ -0,0 +1,74 @@
+#![cfg(feature = "test-sbf")]
+
+mod program_test;
+
+use solana_program_test::tokio;
+
+use program_test::*;
+use spl_governance::state::enums::ProposalState;
+
+#[tokio::test]
+async fn test_execute_complete_proposal_without_transaction() {
+    // Arrange
+    let mut governance_test = GovernanceProgramTest::start_new().await;
+
+    let realm_cookie = governance_test.with_realm().await;
+    let governed_account_cookie = governance_test.with_governed_account().await;
+
+    let token_owner_record_cookie = governance_test
+        .with_community_token_deposit(&realm_cookie)
+        .await
+        .unwrap();
+
+    let mut governance_cookie = governance_test
+        .with_governance(
+            &realm_cookie,
+            &governed_account_cookie,
+            &token_owner_record_cookie,
+        )
+        .await
+        .unwrap();
+
+    let mut proposal_cookie = governance_test
```

Comment:
nit: There is a helper function `with_signed_off_proposal` which combines `Create`, `AddSignatory` and `SingOff` instructions